integrations/object_store/src/store.rs

```diff
@@ -93,17 +92,13 @@ use std::sync::Arc;
 /// }
 /// ```
 pub struct OpendalStore {
-    info: Arc<OperatorInfo>,
     inner: Operator,
 }
 
 impl OpendalStore {
     /// Create OpendalStore by given Operator.
     pub fn new(op: Operator) -> Self {
-        Self {
-            info: op.info().into(),
-            inner: op,
-        }
+        Self { inner: op }
```

Comment:
Hi, this change is by design. Please don't call `info()` inside `Debug`.

Reply:
ok

Reply:
resolved